packages/go_router_builder/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: go_router_builder
 description: >-
   A builder that supports generated strongly-typed route helpers for
   package:go_router
-version: 2.2.2
+version: 2.3.0
```

Comment:
There isn't a new feature, should probably update patch version.

Reply:
Sure, I did it in [chore: Only update patch version](https://github.com/flutter/packages/pull/4524/commits/57f71a960370c0da9b138b0e58d8fe6bce0cdeed)